tests/fe/fe_values_view_30.cc

```diff
@@ -109,7 +109,7 @@ void test (const Triangulation<dim> &tr,
 
   MappingQGeneric<dim> mapping(1);
 
-  const QGauss<dim> quadrature(2);
+  const QGauss<dim> quadrature(fe.degree+2);
```

Comment:
Actually, I just wanted to have the same output for the new and the old code. Since, `project_generic` uses `QGauss<dim> quadrature(fe_degree+2)` this implies to use the same quadrature for the right-hand side.